components/campaign/ChaosToken.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { ImageBackground, StyleSheet, Text, View } from 'react-native';
+import { ChaosTokenType } from '../../constants';
+import ChaosTokenIcon from '../core/ChaosTokenIcon';
+import { iconSizeScale } from '../../styles/space';
+import DeviceInfo from 'react-native-device-info';
+import { COLORS } from '../../styles/colors';
+const SCALE = ((DeviceInfo.getFontScale() - 1) / 4 + 1);
+
+const CHAOS_TOKEN_BACKGROUND = require('../../assets/chaos-token-background.jpg');
+
+interface OwnProps {
+  iconKey?: ChaosTokenType;
+  small?: boolean;
+}
+
+type Props = OwnProps;
+
+export default class ChaosToken extends React.Component<Props> {
+
+  _renderIcon = () => {
+    const { iconKey, small } = this.props;
+    const size = small ? 25 : 50;
+
+    if (iconKey) {
+      return (
+        <Text>
+          <ChaosTokenIcon icon={iconKey} size={size * SCALE * iconSizeScale} color="#fff" fontFamily="Teutonic" />
+        </Text>
+      );
+    }
+  }
```

Comment:
Semicolon at end of the bound function, I believe `npm run build` would complain about this.

Reply:
WebStorm wasn't complaining based on eslint but I'll see what `npm run build` has to say